Idno/Pages/Service/Queues/GC.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Idno\Pages\Service\Queues {
+
+    use Idno\Core\Idno;
+
+    class GC extends \Idno\Common\Page {
+        
+        public function getContent() {
+            
+            Idno::site()->template()->setTemplateType('json');
+            
+            \Idno\Core\Service::gatekeeper();
+            
+            $eventqueue = \Idno\Core\Idno::site()->queue();
+            if (!$eventqueue instanceof \Idno\Core\AsynchronousQueue) 
+                throw new \RuntimeException("Service can't run unless Known's queue is Asynchronous!");
```

Comment:
What does this error message mean? Is there a clearer way we can say this?

Reply:
It actually probably isn't needed at all in this context. Copy/paste error :)